evaluation frequency depending on the evaluation timeframe (#11569)
DataDog/documentation: content/en/monitors/monitor_types/metric.md

* the threshold `on average`, `at least once`, `at all times`, or `in total`
* during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 730 hours (1 month).

> Depending on the evaluation time frame selected, the evaluation frequency changes :
> * `timeframe < 24h` : evaluation is performed every 1 minute
> * `24h < timeframe < 48h` : evaluation is performed every 10 minutes
> * `timeframe > 48h` : evaluation is performed every 30 minutes

Reviewer: ```suggestion
The evaluation frequency changes based on the evaluation time frame you select:

* `timeframe < 24h` : evaluation performs every 1 minute.
* `24h < timeframe < 48h` : evaluation performs every 10 minutes.
* `timeframe > 48h` : evaluation performs every 30 minutes.
```

Author: Commited your suggestions, thanks !

* the threshold `on average`, `at least once`, `at all times`, or `in total`
* during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 730 hours (1 month).

The evaluation frequency changes based on the evaluation time frame you select:

> * `timeframe < 24h` : evaluation performs every 1 minute.
> * `24h < timeframe < 48h` : evaluation performs every 10 minutes.
> * `timeframe > 48h` : evaluation performs every 30 minutes.

Reviewer: ```suggestion
* `timeframe < 24h`: evaluation performs every 1 minute.
* `24h < timeframe < 48h`: evaluation performs every 10 minutes.
* `timeframe > 48h`: evaluation performs every 30 minutes.
```
